app/controllers/annotations_controller.rb

```diff
@@ -26,33 +26,30 @@ def index
       end
     end
 
+  def getAnnotationsForCanvas
```

Comment:
Let's use snake_case instead of camelCase for all new method and variable names.

Reply:
I believe this is one that Roy had implemented. I was hesitant to change it because there is a defined route pointing to this method. I'll update the controller and route and use that standard going forward for any files I touch.